packages/ts/lit-form/src/Validators.ts

```diff
@@ -268,7 +268,8 @@ export class Negative<T> extends AbstractValidator<T> {
   }
 
   override validate(value: T): boolean {
-    return toFloat(String(value)) < 0;
+    const fv = toFloat(String(value));
+    return isNaN(fv) || fv < 0;
```

Comment:
Overall I'd approve these changes but wouldn't it be better to use null check before conversion?

```suggestion
    if (value != null) {
      return toFloat(String(value)) < 0
    }
    
    return true;
```

Reply:
I wanted to tackle `null`, but also `undefined` and `''`.

Reply:
Ah, ok, so I missed the `''` one